.storybook/manager-head.html

```diff
@@ -1,9 +1,9 @@
-<head>
-    <script>
-        if (window.location.pathname === "/storybook") {
-            const script = document.createElement("base");
-            script.href = "/storybook/";
-            document.getElementsByTagName('head')[0].appendChild(script);
-        }
-    </script>
-</head>
+<head>
+    <script>
+        if (window.location.pathname === "/storybook") {
+            const script = document.createElement("base");
+            script.href = "/storybook/";
+            document.getElementsByTagName('head')[0].appendChild(script);
+        }
+    </script>
+</head>
```

Comment:
Did you `cleanup .`? Go for `touch node_modules/.cignore` to save you from trouble